src/Raven.Client/Documents/Operations/CompareExchange/PutCompareExchangeValueOperation.cs

```diff
@@ -47,7 +47,7 @@ public PutCompareExchangeValueCommand(string key, T value, long index, DocumentC
                 _key = key;
                 _value = value;
                 _index = index;
-                _conventions = conventions ?? DocumentConventions.Default;
+                _conventions = DocumentConventions.DefaultConventionsForCompareExchange;
```

Comment:
I think that this will not take into account the casing (if changed from default) in serializer?

Reply:
done